src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const utils = require('./utils');
+const removeUndefined = require('./parse/utils').removeUndefined;
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return removeUndefined({
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
```

Comment:
Here you can see how `sequence` got interpreted as a "setting" here, and ended up in the `specification` for a settings transaction on accident: https://github.com/ripple/ripple-lib/pull/427/files#diff-62ddd0ef536cb84665c26471f1a09fa4L9